src/routes/+page.svelte

```diff
@@ -10,12 +11,17 @@
 	onMount(async () => {
         workspace.addChangeListener(()=>workspace=workspace);
 	});
+    let stx = "<spatext/>"
+    $: if (workspace) {
+        stx = `<spatext>\n${SpatextGenerator.workspaceToCode(workspace)}\n</spatext>`
+    }
 </script>
 
 <!-- <input type='number' bind:value={height}/> -->
 <BlocklyComponent {height} width={1200} {toolbox} bind:workspace/>
 
 {#if workspace}
-<textarea id="blocklyProof" style="height: 320px; width: 600px;" readonly>{SpatextGenerator.workspaceToCode(workspace)}</textarea>
+<textarea id="blocklyProof" style="height: 320px; width: 600px;" readonly>{stx}</textarea>
 <textarea id="blocklyCoq" style="height: 320px; width: 600px;" readonly>{CoqGenerator.workspaceToCode(workspace)}</textarea>
+<SpaTeXt {stx}/>
```

Comment:
Now you have access to a `SpaTeXt` component that takes the string `stx` and spits out pretty (imho) typeset mathematicds.